### DISCUSSION TITLE
List the currency on pricing pages

### DISCUSSION THREAD

**Discussion by troypoulter** [plausible/analytics#1947] [Anything Else]
## Feature Request

Add the currency code, such as `USD`, `EUR` & `GBP`, on the [Pricing page](https://plausible.io/#pricing) and other pay related pages, like subscription management.

**Current page**
![plausible_pricing](https://user-images.githubusercontent.com/19419349/172135950-842d2275-4dc6-426e-bb9f-3319ff53952a.png)

**Mock up**
![image](https://user-images.githubusercontent.com/19419349/172136666-de87706d-dacb-459a-943d-38286abb23c8.png)

## Context

I'm based in Australia and when I went to the Pricing page & subscription management in my account, I saw the base price as $9/month for 10k views.

I was trying to confirm whether the currency was in `AUD` or something different, I had assumed it would be `EUR` based on where the servers are hosted.

It wasn't until after I paid and got the invoice did I see what currency it was in.

![plausible_invoice](https://user-images.githubusercontent.com/19419349/172135777-ca09bc12-a55c-41aa-93af-dee79c32c6d2.png)

I searched around and tried to find related issues/code, came across the discussion https://github.com/plausible/analytics/discussions/144 and though semi-relevant, this is to more display the currency code, not change it based on country.

**Comment by metmarkosaric** [comment]
thanks for the feedback @troypoulter! we do list `€` for Euro zone countries and `£` for the UK. it makes sense to clarify which `$` we're talking about especially in countries such as Australia and Canada which also use the `$` sign as we support `$ USD` only

**Reply by troypoulter** [reply]
Ahh yes that makes sense with € & £. Then putting just the USD when using the $ or even a note on pricing pages that currency is USD would be good!

**Comment by smashingstuff** [comment]
+1 for this. Another point/gotcha for people who are at the point of upgrading... Plausible does not honor the remainder of your trial when you go paid. In other words, if you have 7 days left on your trial when you upgrade, you will lose those.

I'm guessing this would have been a more complex integration with Paddle, but even so, it is a bit annoying.

**Reply by metmarkosaric** [reply]
hi @smashingstuff! yeah, that's correct and you're right on the reasons for it too. we'd like to change this in the future after all the feature work is behind us. for now, we're happy to manually extend the subscription to add the amount of days you've missed out on your trial. do [contact our support please](https://plausible.io/contact). sorry for the annoyance with this!

**Reply by smashingstuff** [reply]
As a quick fix, perhaps it would be a helpful idea to modify the body of your "Your Plausible trial expires next week" email reminder, to explain this to people, and perhaps add a line that says "If you'd rather not upgrade today for this reason, we totally understand, but don't worry, we'll send another reminder the day before your trial ends, so you can upgrade then if you like", assuming you do in fact send such email (which I don't know, I never got to that stage). Just some hopefully helpful feedback. :)

**Comment by hayden-t** [comment]
so is it aud or usd in australia ?

**Reply by metmarkosaric** [reply]
hi @hayden-t! it's USD
